Source/Krypton Components/Krypton Toolkit Suite 2022 - VS2022.sln

```diff
@@ -90,17 +90,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestForm", "TestForm\TestFo
 		{F9E31BD7-42BE-48E8-92CC-74CA1871B4E6} = {F9E31BD7-42BE-48E8-92CC-74CA1871B4E6}
 	EndProjectSection
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "dictionary-en-gb", "dictionary-en-gb", "{CAB40818-55B1-4E1D-AD8E-C94642EA5124}"
```

Comment:
Why have you removed the dictionary from the project ?

Reply:
Do we know what these dictionaries are used for, as they've just only appeared?

Reply:
ReSharper spelling, so that I do not have to keep adding English GB words to the dictionary !

Reply:
I'll add a note in the `readme.md` stating its use